apps/web/components/forgot-password.tsx

```diff
@@ -0,0 +1,142 @@
+import { useState } from "react";
+import Link from "next/link";
+import { Button } from "@documenso/ui";
+import Logo from "./logo";
+import { ArrowLeftIcon } from "@heroicons/react/24/outline";
+import { FormProvider, useForm } from "react-hook-form";
+import { toast } from "react-hot-toast";
+
+interface IForgotPassword {
+  email: string;
+}
```

Comment:
We don't prefix with `I` in Typescript, instead try `ForgotPasswordForm` since that's what we're working with here 😄